src/plugins/unified_search/public/query_string_input/add_filter_popover.tsx

```diff
@@ -69,7 +69,7 @@ const AddFilterPopoverComponent = React.memo(function AddFilterPopover({
         size="m"
         disabled={isDisabled}
         {...buttonProps}
-        style={{ borderTopLeftRadius: 0, borderBottomLeftRadius: 0 }}
+        css={{ borderTopLeftRadius: 0, borderBottomLeftRadius: 0 }}
```

Comment:
To update: Make sure the buttonProps styling is preserved:

```suggestion
        css={{ ...buttonProps.css, borderTopLeftRadius: 0, borderBottomLeftRadius: 0 }}
```